django/contrib/gis/db/backends/mysql/schema.py

```diff
@@ -81,8 +81,7 @@ def create_spatial_indexes(self):
                 self.execute(sql)
             except OperationalError:
                 logger.error(
-                    "Cannot create SPATIAL INDEX %s. Only MyISAM and (as of "
-                    "MySQL 5.7.5) InnoDB support them.",
-                    sql,
+                    f"Cannot create SPATIAL INDEX {sql}. Only MyISAM and InnoDB "
+                    f"support them.",
```

Comment:
```suggestion
                    "support them.",
```
_f-string without any placeholders_

Reply:
Yes, but it's concatenated with a f-string. I'd rather keep it that way, so:
```python
text = (
    f"something with {placeholder}"
    f"pure text without any placeholder"
    f"next part with {placeholder2}"
)
```
instead of
```python
text = (
    f"something with {placeholder}"
    "pure text without any placeholder"
    f"next part with {placeholder2}"
)
```

Reply:
Personally I prefer to have the evidence of strings that contain a placeholder, but I understand that it is an aesthetic choice, so you can ignore my comment.

Reply:
As we're here in this logger message, is `Only MyISAM and InnoDB support them` correct? 
I'm not _aware_ of the differences between Maria and MySQL, but `supports_spatial_index` suggests that `Aria` as a separate storage engine is also supported. Is `Aria` just different nomenclature for the same thing?

Reply:
Good catch :+1: Updated.